src/components/settings/withings-connection.tsx

```
@@ -0,0 +1,355 @@
+          break;
+      }
+      
+      toast({
+        title: 'Connection Failed',
+        description: errorMessage,
+        variant: 'destructive',
+      });
+      // Clear URL params
+      window.history.replaceState({}, '', window.location.pathname);
+    }
+  }, [toast, loadConnectionStatus]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)